Added note about monitors and security (#372)
opensearch-project/documentation-website: _monitoring-plugins/alerting/security.md

After making the change, the monitor now runs with the same permissions as `psantos`, including any [document-level security]({{site.url}}{{site.baseurl}}/security-plugin/access-control/document-level-security/) queries, [excluded fields]({{site.url}}{{site.baseurl}}/security-plugin/access-control/field-level-security/), and [masked fields]({{site.url}}{{site.baseurl}}/security-plugin/access-control/field-masking/). If you use an extraction query to define your monitor, use the **Run** button to ensure that the response includes the fields you need.

> Once a monitor is created, the Alerting plugin will continue executing the monitor, even if the user who created the monitor has write access permissions removed. To stop a monitor, a user with at least `alerting_write_access` permissions must manually disable or delete the monitor. This rule applies to all types of monitors, regardless of destination or any other setting.

Reviewer: We also want to indicate that if alerts are generated from the triggers, notifications will still be sent out if they have been configured. Also may want to note that this includes SNS in the AWS Opensearch as well.

> Once a monitor is created, the Alerting plugin will continue executing the monitor, even if the user who created the monitor has write access permissions removed. To stop a monitor, a user with at least `alerting_write_access` permissions must manually disable or delete the monitor. This rule applies to all types of monitors, regardless of destination or any other setting. If your monitor's trigger has alerts configured, the Alerting plugin continues to send out alerts unless the trigger's action is manually deleted.

Reviewer: `Alerting plugin continues to send out alerts unless the trigger's action is manually deleted`
This should be updated to something along the lines of: `Alerting plugin continues to send out alerts unless the trigger is manually deleted or the monitor is manually deleted/disabled`

We need to specify that the notifications to the destinations will also be sent out irregardless of the destination type.

After making the change, the monitor now runs with the same permissions as `psantos`, including any [document-level security]({{site.url}}{{site.baseurl}}/security-plugin/access-control/document-level-security/) queries, [excluded fields]({{site.url}}{{site.baseurl}}/security-plugin/access-control/field-level-security/), and [masked fields]({{site.url}}{{site.baseurl}}/security-plugin/access-control/field-masking/). If you use an extraction query to define your monitor, use the **Run** button to ensure that the response includes the fields you need.

> Once a monitor is created, the Alerting plugin will continue executing the monitor, even if the user who created the monitor has permissions removed. To stop a monitor, a user with the correct cluster permissions must manually disable or delete the monitor. To disable a monitor, the user must have `cluster:admin/opendistro/alerting/monitor/write`, and to delete a monitor, the user must have `cluster:admin/opendistro/alerting/monitor/delete`. This rule applies to all types of monitors.

Reviewer 2: There's a comma splice in the sentence, "To disable a monitor...". Switching the permissions to a short unordered list might improve readability. For example:

```suggestion
Once a monitor is created, the Alerting plugin will continue executing the monitor, even if the user who created the monitor has their permissions removed. A user can only manually delete or disable a monitor with the correct cluster permissions. This rule applies to all types of monitors. 

- To disable a monitor, the user must have the `cluster:admin/opendistro/alerting/monitor/write` permission.
- To delete a monitor, the user must have the `cluster:admin/opendistro/alerting/monitor/delete` permission.
```

You can also remove the comma splice with a period.

> If your monitor's trigger has notifications configured, the Alerting plugin continues to send out notifications regardless of destination type. To stop notifications, a user must manually delete them in the trigger's actions.

Reviewer 2: I have a voice question. Do we prefer to use "a user" or the less formal "you" when guiding the reader?

Author: We don't have a hard and fast rule about which one to use. In this case, I wanted to use "user" because of the association to roles and permissions, and I didn't want the reader to get confused about what they specifically can do about Alerting permissions.
